.buildkite/premerge.steps.yaml

```diff
@@ -17,6 +17,23 @@ script_runner: &script_runner
   - "scaler_version=2"
   - "working_hours_time_zone=london"
 
+common: &common
+  agents:
+    - "agent_count=1"
```

Comment:
Would 1 agent be sufficient?

Reply:
I believe this specifies how many BK steps may concurrently run on a single "physical" machine. Setting this to 1 in theory helps to make sure build jobs are not failing because other processes are already using fastbuild.